css/secondary-page.styl

```diff
@@ -62,6 +62,7 @@
               background: rgba(255,255,255, 0.2)
               color: rgba(255,255,255,1)
             &.active
+            &[aria-selected=true]
```

Comment:
By the way, I added the aria roles to these because the Projects page is using buttons to filter the projects list, whereas other pages use links for the secondary nav.